code/style.css

```diff
@@ -1,7 +1,210 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+* { box-sizing: 
+  border-box; margin: 0
+ } 
+body {
+  
+  font-family: sans-serif; 
+}
+
+.logo{
+  display: none;
+}
+.navigation{
+  display: flex;
+  justify-content: space-evenly;
+  padding-top: 20px;
+  align-items: center;
+}
+.navigation-link{
+  color: rgb(197, 102, 39);
+ 
+}
+.hero {
+  /* Sizing */
+  width: 100%;
+  height: 100vh;
+
+
+/* Flexbox stuff */
+display: flex;
```

Comment:
Is this just in general?

Reply:
This is in the .hero div styleing the hero section :D
```
.hero {
  /* Sizing */
  width: 100%;
  height: 100vh;


/* Flexbox stuff */
display: flex;
   justify-content: center;
   align-items: center;

 /* Text styles */
 text-align: center;
}
```